documentatie/flowcharts/data-entry-forms.md

```diff
@@ -61,28 +64,24 @@ flowchart TD
 
     %% elements
     flow-start([start])
+    flow-done([done])
 
     render-empty-fields([render with empty fields])
     render-cached-data([render with cached data])
-    flow-end([end])
 
     render-submitted-data([render with submitted data])
 
     show-error([show error])
-    hide-checkbox-accepted(["hide checkbox
-        warnings accepted"])
+    hide-checkbox-accepted(["hide checkbox warnings accepted"])
     show-warning([show warning])
-    show-checked-accepted(["show checked checkbox
-        warnings accepted"])
-    show-unchecked-accepted(["show unchecked checkbox
-        warnings accepted"])
+    show-checked-accepted(["show checked checkbox warnings accepted"])
+    show-unchecked-accepted(["show unchecked checkbox warnings accepted"])
```

Comment:
These were split over two lines, because that carries over into the rendered version, making the containing box narrower and thus the chart more compact. You prefer the wider version?

Reply:
It didn't seem to make a difference yesterday, but going over it again now it does make a difference.

I've re-added the line breaks with `\n` so that my IDE doesn't complain.